packages/hardhat/contracts/strategy/CrabStrategy.sol

```diff
@@ -824,6 +823,7 @@ contract CrabStrategy is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Ownab
      * @return true if hedging is allowed
      */
     function _isPriceHedge(uint256 _auctionTriggerTime) internal view returns (bool) {
+        require( _auctionTriggerTime > timeAtLastHedge, "Trigger time must be after time at last hedge");
```

Comment:
@haythem96 is it bad for a view function to revert?

We expose checkPriceHedge which is an external view function.

Reply:
Yeah IMO `external view` should always return a result, in this case either True or False

Reply:
I guess to follow the initial design, u can add `if` statement inside `_isPriceHedge` in the beginning:
```
if (_auctionTriggerTime > timeAtLastHedge) return false;
```